README.md

```diff
@@ -1350,6 +1350,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Michael Akintomo (@MadeByM1KE)](https://github.com/Makintomo04)
   - [Michael Godfrey](https://github.com/michagodfrey)
   - [Michael Liendo](https://github.com/Michael-Liendo)
+    [Michael Ochei](https://github.com/MichaelFED)
```

Comment:
```suggestion
  - [Michael Ochei](https://github.com/MichaelFED)
```

Reply:
I have correctly added the "-", seems the merge was automatic??